autoware.repos

```diff
@@ -49,6 +49,10 @@ repositories:
     type: git
     url: https://github.com/tier4/pointcloud_to_laserscan.git
     version: tier4/main
+  universe/external/scenario_simulator_v2_msgs:
+    type: git
+    url: https://github.com/tier4/scenario_simulator_v2_msgs.git
```

Comment:
I'm not sure whether we should add TEIR IV's simulator-specific messages here.
I'd like to get reviews from @xmfcx @mitsudome-r @yukkysaito.

Why not adding the messages to https://github.com/autowarefoundation/autoware_msgs (Autoware's official messages) or https://github.com/tier4/tier4_autoware_msgs (TIER IV's proposal messages)?

Also, have you considered releasing the messages to the ROS buildfarm? Please share the discussion results.
In the cases of `autoware_msgs` and `tier4_autoware_msgs`, since they are still under construction, we're not planning to release them yet.

Reply:
@kenji-miyake thank you 👍 
I do not think it is a good idea to add it since it is a test msg inside TIER IV.

If it is to be added, I think it would be better to make a proposal to AWF whether or not to introduce a scenario simulator.

Reply:
@kenji-miyake @yukkysaito cc: @yamacir-kit  
Thank you for the comments.
Let me confirm whether [scenarios_simulator_v2](https://github.com/tier4/scenario_simulator_v2) is being used as AWF's evaluator for the evaluation.
If scenario_simulator_v2 is tier4's specific repository, I think `scenario_simulator_v2_msgs` should be in https://github.com/tier4/tier4_autoware_msgs.

@mitsudome-r 
Do you know whether scenario_simulator_v2's position is AWF evaluator or tier4's evaluator?
(I heard scenario_simulator_v2 is being used by AWF's project but not sure if it's used as AWF's evaluator)

Reply:
We are using TIER IV's evaluator in AWF as well.

However, that does not mean it has to go into autoware_msgs. If you want to make this a core interface between Autoware Foundation and simulator in general, it should go into autoware_msgs.(i.e., if you want to add this to AD API or component interface)

If this topic is only meant to be used by TIER IV, then it should be merged into tier4_autoware_msgs.

In either case, I would like to see more description on what/how this message is used for. You wrote that this message is needed to merge this [pr](https://github.com/autowarefoundation/autoware.universe/pull/2514), but description in the PR doesn't explain what you want to solve. There is no design discussion shared with us to decide why we should add this message to Autoware.

Reply:
@mitsudome-r 
I described how the msg is used for in this [pr](https://github.com/autowarefoundation/autoware.universe/pull/2514,). Could you check the description?

Reply:
@mitsudome-r @kenji-miyake @yukkysaito cc: @yamacir-kit
We have a message for simulation named `tier4_simulation_msgs` created previously in tier4_autoware_msgs repository, so making a draft [PR](https://github.com/tier4/tier4_autoware_msgs/pull/73), but I'm not sure if it's a good idea.
Could you give some comments where we should add those msgs?

Reply:
@kyoichi-sugahara What's your concern? I think it looks good. And it would be good if TIER IV could propose the message definitions to the AWF with more concrete use cases.